### ISSUE TITLE
[Request] Progress bar/status for --migrate-players

### ISSUE THREAD

**Issue by Megaf** [minetest/minetest#7697]
##### Issue type

 - Feature request
 
##### Minetest version
```
Stable 0.4 and 5.x
```

##### Summary
Hi folks. We have recently added to Minetest the amazing new player storage, now in sqlite3. Which is awesome! And better than that minetestserver is able to migrate existing plain text database to the new one.

Now please, can you add a progress bar or something that gives an insight on what's going on or a percentage or something?

My very server has been running the migrate players for 31 hours already and I have no idea how much more it will take.
Thanks!

**Comment by paramat** [comment]
:+1: Seems much needed then

**Comment by Megaf** [comment]
![image](https://user-images.githubusercontent.com/6201512/45036090-72d2ec80-b053-11e8-9a6c-a1434bf7a0d3.png)
42 hours, still going.

**Comment by paramat** [comment]
From IRC:
16:58 Megaf          also, minetestserver will not provice any indication that is actually working
16:58 Megaf          provide*
16:59 sfan5          it will, it logs every user that is migrated
16:59 Megaf          log where?
16:59 sfan5          actionstream, so your terminal

So issue can be closed?

**Comment by Megaf** [comment]
Nope, I there was any status there I wouldn't be opening this issue, would?
As you can see on my screenshot, there's is nothing there. **42 hours running, no messages, no debug, not even with --trace.**

Issue stays open until feature is implemented and if it exists, then improved.

**Comment by rubenwardy** [comment]
It's probably another issue which makes you stuck at 0%. A message before any work was done would show this

**Comment by paramat** [comment]
What i meant was that progress messages are implemented already as sfan5 has confirmed, but you didn't see any due to the problem you have, your migration probably halted before the first player was migrated. So the request is already done, the real issue here is something else and covered by #7701 
However showing a message before any migration might be helpful as rubenwardy suggests.

**Comment by paramat** [comment]
+1 for rubenwardy's suggestion.

**Comment by paramat** [comment]
Feature is already present, changing topic title to be for some kind of initial message, as suggested.
